game-core/src/main/java/org/triplea/util/LocalizeHtml.java

```diff
@@ -62,23 +46,9 @@ static String localizeImgLinksInHtml(final String htmlText, final ResourceLoader
     return result.toString();
   }
 
-  private static String getLocalizedLink(final String link, final ResourceLoader loader) {
-    // remove full parent path
-    final String imageFileName = link.substring(Math.max(link.lastIndexOf("/") + 1, 0));
-
-    // replace when testing with: "REPLACEMENTPATH/" + imageFileName;
-    final String firstOption = ASSET_IMAGE_FOLDER + imageFileName;
-    URL replacementUrl = loader.getResource(firstOption);
-
-    if (replacementUrl == null) {
-      log.error(String.format("Could not find: %s/%s", loader.getMapName(), firstOption));
-      final String secondFallback = ASSET_IMAGE_FOLDER + ASSET_IMAGE_NOT_FOUND;
```

Comment:
Is the fallback image never used?

Reply:
Yes, it's never used, the file 'notFound.png' is not part of the game engine. A map could have it, but none do.